apps/backend/src/libs/hooks/check-user-permissions.hook.ts

```diff
@@ -16,6 +16,14 @@ const checkUserPermissions = (routePermissions: string[]): APIPreHandler => {
 			});
 		}
 
+		const resourceId = (params as { id?: string }).id;
+
+		if (url.includes(APIPath.USERS) && Number(resourceId) === user.id) {
```

Comment:
How about adding a variable like `isCurrentUserRoute`,  `hasFreeAccess` or something for the condition? Just to keep semantic

Reply:
Thanks, nice notice. I've changed this, however decided on a little different names.